dtc/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Andy Gross <andy.gross@linaro.org>
+
+pkgname=dtc
+pkgver=1.4.4
```

Comment:
Is there a way to start with master instead of a released version? 1.4.4 won't build on MSYS2.